### PR TITLE
add debuggable webassembly test project

### PR DESCRIPTION
**Pull request by dibarbet**

### CODE REVIEW

File: src/OmniSharp.WebAssembly.Driver/Program.cs [hunk 1]
```diff
@@ -25,82 +25,7 @@ public static void Main(string[] args)
...
-            }
-        }
-    }
-
-    class ConsoleEventEmitter : IEventEmitter
-    {
-        public void Emit(string kind, object args)
-        {
-            Console.WriteLine($"[{kind}]{args}");
+            return await Host.InitializeAsync(compilerLogBytes);
```
Comment (dibarbet) [inline src/OmniSharp.WebAssembly.Driver/Program.cs:28]: moved - the blazor test wasm project cant depend on this blazor wasm project

File: browser.html [hunk 2]
```diff
@@ -1,24 +0,0 @@
-<!-- Example on consuming the library as global script. -->
-<!-- Open the .html file in a web browser to test. -->
-
-<!DOCTYPE html>
-
-Open console (Ctrl+Shift+I) to see the output.
-
-<!-- Import as a global 'dotnet' object via script tag. -->
-<script src="artifacts/publish/OmniSharp.WebAssembly.Driver/net6.0/dotnet.js"></script>
```
Comment (dibarbet) [inline browser.html:9]: broken and not useful now that the other debug proejct exists

File: tests/WebAssembly/TestWithWebAssembly/Pages/Index.razor [hunk 3]
```diff
@@ -0,0 +1,92 @@
+﻿@page "/"
+@using OmniSharp.WebAssembly
+@using System.Text.RegularExpressions
+@using System.Web
+
+<PageTitle>Index</PageTitle>
+
+<h1>Hello, world!</h1>
```
Comment (dibarbet) [inline tests/WebAssembly/TestWithWebAssembly/Pages/Index.razor:8]: just a simple page that allows you to upload a file and then prints out the console output

File: tests/WebAssembly/TestWithoutWebAssembly/Program.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TestWebAssemblyDriver;
+
+internal class Program
+{
+    static void Main(string[] args)
```
Comment (dibarbet) [inline tests/WebAssembly/TestWithoutWebAssembly/Program.cs:5]: this allows us to test w/out webassembly if you need to

**Review by 333fred — APPROVED** [review]

**Pull request merged**